### DISCUSSION TITLE
Questions about adding a formula to homebrew core

### DISCUSSION THREAD

**Discussion by ghost** [Homebrew/discussions#3458] [Tap maintenance and Homebrew development]
We are developing a new open source package and we would like to see if we can add the formula to homebrew core.
Our package has dependencies such as lima, qemu etc, and we need to apply some patches to them and also pin them to specific version instead of blindly take the latest. So, we have our own way to maintain those dependencies and manage their version in our own storage. Is this acceptable by homebrew-core? Thanks.

<img width="961" alt="image" src="https://user-images.githubusercontent.com/10912460/180508822-5b6e993a-dc87-43f5-916c-bdffd84c9e5c.png">

**Comment by SMillerDev** [comment]
Unfortunately those are all listed as rejection reasons in https://docs.brew.sh/Acceptable-Formulae
